### PR TITLE
fix(rn): Remove hermesc envs if hermes is disabled

### PR DESCRIPTION
**Pull request by krystofwoldrich**
If hermesc path is in envs and exists react native tooling assumes hermes is enabled.

Sentry cli puts itself on the path therefore react native tooling always assumes hermes is enabled.

### Todo:
- [x] Test with multiple RN versions.

### CODE REVIEW

File: src/commands/react_native/xcode.rs [hunk 1]
```diff
@@ -263,16 +271,24 @@ pub fn execute(matches: &ArgMatches) -> Result<()> {
         // With that we we then have all the information we need to invoke the
         // upload process.
         } else {
-            let rv = process::Command::new(&script)
+            let mut command= process::Command::new(&script);
```
Comment (loewenheim) [inline src/commands/react_native/xcode.rs:274]: Please run `cargo fmt` on the project as well.

**Review by loewenheim — CHANGES_REQUESTED** [review]

File: src/commands/react_native/xcode.rs (hunk 1 above)
Comment (krystofwoldrich) [reply src/commands/react_native/xcode.rs:274]: I did, but no changes emitted.

I fixed one extra empty line manually.

Comment (loewenheim) [reply src/commands/react_native/xcode.rs:274]: Hm, that's odd. It doesn't do anything for me either, but my editor formats this correctly. Do you mind if I push my formatting changes?

Comment (krystofwoldrich) [reply src/commands/react_native/xcode.rs:274]: I don't mind. Go ahead.

**Review by loewenheim — APPROVED** [review]

**Review by loewenheim — APPROVED** [review]

**Pull request merged**